base/strings/search.jl

```diff
@@ -158,15 +158,26 @@ findfirst(pattern::AbstractVector{<:Union{Int8,UInt8}},
           A::AbstractVector{<:Union{Int8,UInt8}}) =
     _search(A, pattern, firstindex(A))
 
-# AbstractString implementation of the generic findnext interface
-function findnext(testf::Function, s::AbstractString, i::Integer)
-    i = Int(i)
+"""
+    findnext(predicate::Function, s::AbstractString, start::Integer)
+
+Returns the index `i` of the first character after `start` for which
```

Comment:
```suggestion
Returns the index `i` of the first character whose index is `≥ start` for which
```

Reply:
Oops, good catch :eyes: Fixed in 445deeaf3b (along with `findprev()`).